.env.sample

```
@@ -1,3 +1,8 @@
 REACT_APP_PRICES_AUTH_TOKEN=token
 REACT_APP_PRICES_ENDPOINT=http://localhost:5000/cse115a/us-central1/getPrices
+
+
+REACT_APP_GOOGLE_MAPS_API_KEY=token
+
 REACT_APP_GOOGLE_MAPS_PLACES_API_KEY=token
+
```

NIT: all this whitespace is unnecessary. Must have been inserted when dealing with the merge conflicts.  
```suggestion
REACT_APP_GOOGLE_MAPS_API_KEY=token
REACT_APP_GOOGLE_MAPS_PLACES_API_KEY=token

```
Make sure that end line is there